src/Events/Authorization/PublishScopeOrAccessTokenHandler.cs

```diff
@@ -0,0 +1,51 @@
+﻿using System.Threading.Tasks;
+
+using Altinn.Platform.Events.Configuration;
+
+using Microsoft.AspNetCore.Authorization;
+using Microsoft.AspNetCore.Http;
+using Microsoft.Extensions.DependencyInjection;
+
+namespace Altinn.Platform.Events.Authorization
+{
+    /// <summary>
+    /// Authorization handler for the <see cref="PublishScopeOrAccessTokenRequirement"/> class.
+    /// </summary>
+    public class PublishScopeOrAccessTokenHandler : AuthorizationHandler<PublishScopeOrAccessTokenRequirement>
```

Comment:
I would prefer these two policies to be separate. Is that causing issues?

Reply:
Read a bit more about it. We must implement this as one policy, but it should be possible for one Policy to use two handlers. Might be something to look into?

Reply:
Spent quite some time before I landed on this solution. Couldn't find anything else feasible to handle the OR condition. So I would need a concrete example or a commit to the PR to handle the implementation differently. 

That being said, there are two handlers being used. We are using the handler made available to the authorization service. Lines 33 and 41 call existing policies registered in Program.cs

Reply:
I would suggest we keep it as is. Keeps us from adding an extra class and introducing more boilerplate code while also making it less obvious how the access is granted. One would need to check all handlers to see which requirement they are related to instead of it being 1:1 between requirement and handler.